charts/backstage/ci/affinityString-values.yaml

```diff
@@ -0,0 +1,21 @@
+backstage:
+  affinity: |-
```

Comment:
Based on `values.yaml`, the `affinity` type is an object, not a string. This will not play nicely with a schema definition (which we currently don't have, but want to have). Let's support an object value only.

Helm will complain with:

```
$ helm template -f charts/backstage/ci/affinityString-values.yaml charts/backstage
coalesce.go:220: warning: cannot overwrite table with non table for backstage.backstage.affinity (map[])
coalesce.go:220: warning: cannot overwrite table with non table for backstage.backstage.affinity (map[])
...
```